build.sbt

```diff
@@ -32,4 +32,4 @@ lazy val nuker = (project in file("nuker"))
   .enablePlugins(JavaAppPackaging)
   .dependsOn(core % "test->test;compile->compile")
 
-Test / parallelExecution := false
+concurrentRestrictions in Global += Tags.limit(Tags.Test, 1)
```

Comment:
@LizBaldo added this..this slows down tests run a bit...but I think it's ok since this repo is fairly small. The benefit is `sbt resourceValidator/test` will run successfully now :)